packages/vm/tests/tester/index.ts

```diff
@@ -91,6 +91,25 @@ async function runTests() {
   runnerArgs.value = argv.value // GeneralStateTests
   runnerArgs.debug = argv.debug // BlockchainTests
 
+  /**
+   * Edit the forkConfig string to ensure it works with RegEx (escape + characters)
```

Comment:
suggestion for clarity
```suggestion
   * Modify the forkConfig string to ensure it works with RegEx (escape `+` characters)
```